src/endpoints/http/collections/export/lib/archive.js

```diff
@@ -0,0 +1,16 @@
+import yazl from 'yazl';
+
+/**
+ * Gets the size of each collection requested to export.
+ *
+ * @param {Stream} collections - An array of streams containing each collection's documents.
+ * @returns {Stream} - Stream containing zip entries for each collection.
+ */
+export default function archive(collections=[]) {
```

Comment:
I think it's ok to always send the zip file. 2 reasons;

1. keep the API consistent
2. Even 1 collection could be very large. Zip it will save some bandwidth and speed up download time.